info.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="author" content="Chingu Voyage 40 Toucans Team 10" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+		<link rel="stylesheet" href="style.css" />
+		<title>Vite App</title>
+	</head>
+	<body>
+		<header>
+			<nav>
+				<ul class="nav-container">
+					<li class="nav-item">
+						<a href="/" class="nav-link">My notes</a>
+					</li>
+					<li class="nav-item">
+						<a href="/faq.html" class="nav-link">FAQ</a>
+					</li>
+					<li class="nav-item">
+						<a href="/theme.html" class="nav-link">Theme</a>
+					</li>
+					<li class="nav-item active">
+						<a href="/info.html" class="nav-link">Info</a>
+					</li>
```

Comment:
Cannot GET link

I think it would be better to change links like this.
/faq.html -> ./faq.html